scripts/install/setup.sh

```diff
@@ -1,21 +1,75 @@
 #!/usr/bin/env bash
 
-bold() {
-  echo ". $(tput bold)" "$*" "$(tput sgr0)";
-}
-
 err() {
   echo "$*" >&2;
 }
 
-~/spinnaker-for-gcp/scripts/manage/check_duplicate_dirs.sh || exit 1
-~/spinnaker-for-gcp/scripts/manage/check_git_config.sh || exit 1
+[ -z "$REPO_PATH" ] && REPO_PATH="$HOME"
```

Comment:
I find `[ -n "$REPO_PATH" ] || REPO_PATH="$HOME"` a bit more readable, but don't feel too strongly about it (if you think the style you've used is more idiomatic for this use case).